src/graph/graph.c

```diff
@@ -12,7 +12,32 @@
 #include "../GraphBLASExt/GxB_Delete.h"
 #include "../util/rmalloc.h"
 
-/*========================= Synchronization functions ========================= */
+static GrB_BinaryOp _graph_edge_accum = NULL;
+
+/* ========================= GraphBLAS functions ========================= */
+void _edge_accum(void *_z, const void *_x, const void *_y) {
+    EdgeID *z = (EdgeID*)_z;
+    const EdgeID *x = (const EdgeID*)_x;
+    const EdgeID *y = (const EdgeID*)_y;
+
+    EdgeID *ids;
+    /* Single edge ID,
+     * switching from single edge ID to multiple IDs. */
+    if(SINGLE_EDGE(*x)) {
+        ids = array_new(EdgeID, 2);
```

Comment:
I see a corresponding free in `Graph_DeleteEdge` - will relation map entries always be cleared through that one function?

Reply:
I believe so, yes.